wolfcrypt/src/asn.c

```diff
@@ -13121,6 +13122,36 @@ int wc_EncodeName(EncodedName* name, const char* nameStr, char nameType,
     return idx;
 }
 
+/* canonical encoding one attribute of the name (issuer/subject)
+ * call we_EncodeName_ex with CTC_UTF8 for email type
```

Comment:
"we_EncodeName_ex" should be "wc_EncodeName_ex"

Reply:
Thanks.Fixed